hack/build.sh.d/kn-plugin-workflow.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+readonly quarkus_version="2.7.6.Final-redhat-00006"
```

Comment:
And of course check the Quarkus release version, pls. 

 @ederign @ljmotta

/cc @vyasgun

Reply:
@dsimansk I'm waiting for the new RHBQ version number to update this, do you know until when I could send a PR changing this version? Thanks.